plugins/css-blank-pseudo/src/browser.js

```diff
@@ -108,39 +114,71 @@ export default function cssBlankPseudoInit(opts) {
 
 	const handler = generateHandler(options.replaceWith);
 
-	document.body.addEventListener('change', handler);
-	document.body.addEventListener('input', handler);
-
-	// observe value changes on <input>, <select>, and <textarea>
-	const window = (document.ownerDocument || document).defaultView;
+	if (document.body) {
```

Comment:
if the activation is called in the html head `document.body` will still be `undefined`